app/controllers/thread_controller.php

```diff
@@ -0,0 +1,76 @@
+<?php 
+class ThreadController extends AppController            
+{
+    public function index()                        
+    {
+        $threads_per_page = 15; 
+        $pageOne = 1;
```

Comment:
Please follow coding standards on variable names.
Also, avoid magic numbers. This should be a const value instead (and then please follow coding standard for const).